Bottom: d149a8c82ec5808b46cee0ee858e5771b9f13517
Top:    29fdce8ff347d12b7922de2da224ed2905974ae3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-18 20:46:02 +0000

Refresh of make-lesstif_hid-struct-static

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 341e52c..5261ee4 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -56,7 +56,7 @@ typedef struct hid_gc_struct
   char erase;
 } hid_gc_struct;
 
-HID lesstif_hid;
+static HID lesstif_hid;
 static HID_DRAW_API lesstif_graphics;
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented GUI function %s\n", __FUNCTION__), abort()
